src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -74,6 +74,7 @@ const CrossPodQueryForm = () => {
     margin: '20px 0'
   };
 
+  /* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Added these disable notes to code sections that break the rule, but require a larger refactor than we can do in this PR. We should follow up in separate PRs and remove these comments.

Reply:
Yeah for this rule we'll probably have to refactor most of the labels with htmlFor.

In the case of label/DocumentSection however I think we can look more into the case where the label and input components are custom components. (https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md#case-my-label-and-input-components-are-custom-components)

If it's applicable, I think we could try this.